README.md

```diff
@@ -154,29 +159,27 @@ python -m unittest moabb.tests
 
 once it is installed.
 
-### Run the Moabb
+### Use Moabb
 
-```
-python -m moabb.run --verbose
-```
+You could check our [tutorials](./tutorials) and [examples](./examples/) to see how you
+could use MOABB in your project.
 
-## <a name="docs"></a> Documentation:
+## Documentation
```

Comment:
This section contains only one link. Maybe we could insert that link directly to Table of contents?

Reply:
I agree, this not really useful to have only one link